src/stores/Create.ts

```diff
@@ -169,69 +275,63 @@ export const getInitialStateData = (
     CreateEntityStore,
     | 'details'
     | 'endpointConfig'
+    | 'endpointConfigErrors'
     | 'connectors'
     | 'formState'
     | 'endpointSchema'
+    | 'resourceSchema'
     | 'collections'
+    | 'currentCollection'
     | 'resourceConfig'
+    | 'resourceConfigErrors'
     | 'messagePrefix'
     | 'resourceConfigHasErrors'
     | 'endpointConfigHasErrors'
     | 'detailsFormHasErrors'
     | 'collectionsHasErrors'
     | 'isIdle'
     | 'isActive'
+    | 'hasErrors'
 > => {
     return {
         isIdle: true,
         isActive: false,
+        hasErrors: true,
 
         messagePrefix,
         details: initialCreateStates.details(),
-        detailsFormHasErrors: false,
+        detailsFormHasErrors: true,
```

Comment:
Except for edit... forms should default to having errors as we need users to fill stuff out.